app/src/main/res/layout/activity_main_widgets_bill.xml

```diff
@@ -0,0 +1,171 @@
+        android:layout_alignParentStart="true"
+        android:text="Bill Amount" />
+
+    <EditText
+        android:id="@+id/editAmount"
+        android:gravity="center"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:inputType="numberDecimal"
+        android:layout_toRightOf="@+id/labelAmount" />
+
+    <TextView
+        android:id="@+id/labelPercent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignBaseline="@+id/textView2"
+        android:layout_alignBottom="@+id/textView2"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true"
+        android:text="Percent" />
+
+    <TextView
+
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignLeft="@+id/editAmount"
+        android:layout_alignStart="@+id/editAmount"
+        android:layout_below="@+id/editAmount"
+        android:text="15%" />
+
+    <SeekBar
+        android:id="@+id/seekBarPercent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_below="@+id/editAmount"
+        android:layout_marginLeft="18dp"
+        android:layout_marginStart="18dp"
+        android:layout_toRightOf="@+id/textView2" />
+
+    <TextView
+        android:id="@+id/labelTip"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/seekBarPercent"
+        android:text="Tip" />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_below="@+id/labelPercent"
+        android:gravity="center"
+        android:text="0" />
+
+    <TextView
+        android:id="@+id/labelTotal"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/labelTip"
+        android:text="Total" />
+
+    <TextView
+        android:id="@+id/totalTextView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignBaseline="@+id/labelTotal"
+        android:layout_alignBottom="@+id/labelTotal"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true"
+        android:gravity="center"
+        android:text="0" />
+
+    <TextView
+        android:id="@+id/labelDevice"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/labelTotal"
+        android:gravity="center_horizontal"
+        android:text="__________________________" />
```

Comment:
pls delete this TextView and use "View" instead of this

like bellow:
```xml
       <View
         android:id="@+id/labelDevice"
         android:layout_width="match_parent"
         android:layout_height="@dimen/dp_1"
         android:layout_below="@id/labelTotal" />
```